feature_engine/datetime/datetime.py

```diff
@@ -168,15 +169,13 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         if self.missing_values == "raise":
             _check_contains_na(X, self.variables_)
 
-        # convert detected datetime variables
-        X = pd.concat(
+        # convert datetime variables
+        datetime_df = pd.concat(
             [
```

Comment:
Take it you don't want the features converted into datetime in the original dataset. Okay